about.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;900&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <nav class="Navbar">
+    <div class="nav-signin bg-dark d-flex hideOnMobile-f">
+      <ul class="nav-ul gap-close">
+        <li><a href="#" class="fs-medium"><i class="fa fa-facebook fc-white fs-medium"></a></i></li>
+        <li><a href="#" class="fs-medium"><i class="fa fa-twitter fc-white fs-medium"></a></i></li>
+        <li><a href="#" class="fc-white fs-small no-style">Logout</a></li>
+        <li><a href="#" class="fc-white fs-small no-style">Mypage</a></li>
+        <li><a href="#" class="fc-white fs-small no-style">Explore GB</a></li>
+      </ul>
+    </div>
+    <div class="nav-items d-flex align-center">
+      <button type="button" id="Menu-button">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </button>
+      <div class="nav-brand">
+        <a href="index.html"><img src="./images/Summer-Camp.jpg" class="hideOnMobile-f logo" alt="Summer Camp Logo"></img></a>
+      </div>
+      <div class="nav-links d-right" id="nav-options">
+        <ul class="nav-ul Mobile-Menu gap-wide" id="mobile-Menu">
+          <li class='d-right close-menu'><a href="#" class="hideOnDesktop-I no-style Dark-Blue" id="closemenu">&times;</a></li>
+          <li><a class="Dark-Blue fs-medium  no-style" href="index.html">Home</a></li>
+          <li><a class="Dark-Blue fs-medium  no-style" href="index.html#Fixtures">Fixtures</a></li>
+          <li><a class="Dark-Blue fs-medium  no-style" href="index.html#Guest-Container">Partners</a></li>
+          </ul>
+      </div>
+    </div>
+  </nav>
+
+  <section class="About-Headline">
+    <!-- <p class="Headline-quote">“A younger me needed to climb every mountain. The older me says they photograph better from the bottom. Anonymous”</p> -->
+    <p class="Headline-greeting">Hello! Adventure Lovers</p>
+    <h1 class="h1-mobile About-Headline-title">Summer Camp and Mountain Climbing</h1>
```

Comment:
- [ ] On this about.html page, we already have an h1 on this line, as per HTML best practice, we are asked to have only one h1 tag for an application, in your case, you have more than 2 h1 tags, please remove and change them to h2 or h3  all the other h1 tags apart from this on line 41